repolint.json

```diff
@@ -0,0 +1,54 @@
+{
+    "axioms": {
+      "linguist":"language",
+      "licensee":"license",
+      "packagers":"packager"
+    },
+    "rules": {
+      "all": {
+        "apache-license-file:file-contents": ["error", {"files": ["LICENSE*"], "content": "Apache License.*Version 2.0", "fail-on-non-existent": true}],
+        "code-of-conduct-file:file-contents": ["error", {"files": ["CODE_OF_CONDUCT*"], "content": "https://wiki.hyperledger.org/community/hyperledger-project-code-of-conduct", "fail-on-non-existent": true}],
+        "security-file-matches:file-contents": ["error", {"files": ["SECURITY.md"], "content": "https://wiki.hyperledger.org/display/SEC/Defect.Response", "fail-on-non-existent": true}],
+        "readme-file-exists:file-existence": ["error", {"files": ["README.md", "README"]}],
+        "readme-references-license:file-contents": ["error", {"files": ["README.md", "README"], "content": "license", "flags": "i"}],
+        "maintainers-file-exists:file-existence": ["error", {"files": ["MAINTAINERS.md", "MAINTAINERS.rst"]}],
+        "contributing-file-exists:file-existence": ["error", {"files": ["CONTRIBUTING.md"]}],
+        "changelog-file-exists:file-existence": ["error", {"files": ["CHANGELOG.md"]}],
+        "test-directory-exists:directory-existence": ["error", {"directories": ["**/test*", "**/specs", "**/**_test.go"], "nocase": true}],
+        "integrates-with-ci:file-existence": [
+            "error",
+            {
+              "files": [
+                "circle.yml", 
+                ".circleci/config.yml", 
+                "ci/azure-pipelines.yml",
+                ".ci/azure-pipelines.yml",
+                "Jenkinsfile",
+                "Jenkinsfile.ci",
+                "Jenkinsfile.cd"
+              ]
+            }
+          ],
+
+        "notice-file-exists:file-existence": ["warning", {"files": ["NOTICE*"]}],
+        "source-license-headers-exist:file-starts-with": ["warning", {"files": ["**/*.js", "!node_modules/**"], "lineCount": 5, "patterns": ["Copyright", "License"], "flags": "i"}],
```

Comment:
The `source-license-headers-exist` check is looking for javascript files instead of `.go` files. This check also checks the `vendor` directory which needs to be excluded, haven't looked at the doc to see if there is a way to exclude this.

Reply:
You will also need to exclude directories containing `mock` in them since these are generated.

It also doesn't appear to be reporting correctly, I can see many files it flags as invalid, but contain "Copyright" on the second line.

Reply:
5 lines also isn't enough to capture the license in files that contain a go build constraint as the first line of the file